github/event.go

```diff
@@ -52,6 +52,8 @@ func (e *Event) ParsePayload() (payload interface{}, err error) {
 		payload = &DeploymentEvent{}
 	case "DeploymentStatusEvent":
 		payload = &DeploymentStatusEvent{}
+	case "DiscussionCommentEvent":
+		payload = &DiscussionCommentEvent{}
```

Comment:
The test for the switch cases are auto generated by `go generate ./...`. I am not able to understand what do you mean by this? Do you want me to add a small test for each case explicitly so, if someone changes struct name by mistake gets informed for the same??

Reply:
Please correct me if I am wrong, but I did not find any tests for the switch/case, neither manual nor generated.

> Do you want me to add a small test for each case explicitly so, if someone changes struct name by mistake gets informed for the same??
It would be nice if you could add this. Then anybody can add more cases in the future.

Reply:
I saw all a file called `git-accessors_test.go` that was auto-generated, it contained tests for all the payload attributes.

Reply:
>It would be nice if you could add this. Then anybody can add more cases in the future.
Will try to do this. I think it's better to add these tests because auto-generated tests will automatically changed as per the changes done by another contributor, which is not the behaviour we want.